src/definitions/modules/dropdown.less

```diff
@@ -616,13 +616,22 @@ select.ui.dropdown {
         .ui.ui.selection.dropdown@{notInverted}:has(> input:focus) {
             border-color: @selectionFocusBorderColor;
             box-shadow: none;
-            & > i.icon {
+            & > i.icon:not(.remove) {
                 opacity: @selectionIconFocusOpacity;
             }
```

Comment:
This block can now be removed as it is doubled by your new selector block below. And as the check for notInverted doesnt seem to be relevant (your new selector overrides it anyway and the jsfiddle works fine)

Reply:
Thanks. Fixed.